homeassistant/components/sensor/websocket_api.py

```diff
@@ -36,3 +42,19 @@ def ws_device_class_units(
             key=lambda s: str.casefold(str(s)),
         )
     connection.send_result(msg["id"], {"units": convertible_units})
+
+
+@callback
+@websocket_api.websocket_command(
+    {
+        vol.Required("type"): "sensor/numeric_device_classes",
+    }
+)
+def ws_numeric_device_classes(
+    hass: HomeAssistant, connection: websocket_api.ActiveConnection, msg: dict[str, Any]
+) -> None:
+    """Return supported units for a device class."""
```

Comment:
The docstring doesn't match what we return from this command. We return the device classes, not the units.